### PR TITLE
Create s1ow.json

### PR DESCRIPTION
**Pull request by akk1to**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.  

## Website Link/Preview

https://h4iph0ngm1enb4c.github.io/Web

### CODE REVIEW

**Comment by akk1to** [comment]
I'll ask the author to prove that I register for him hehe

**Comment by H4iPh0ngM1enB4c** [comment]
Yup I'm H4iPh0ngM1enB4c here. I asked akk1to to register the domain for me because I don't know how to make one lmao

File: domains/s1ow.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    "owner": {
+        "username": "H4iPh0ngM1enB4c",
+        "email": "noobdownload2@gmail.com",
+        "discord": "1027559943462727740"
+    },
+    "record": {
+        "A": [
+          "185.199.108.153",
+          "185.199.110.153",
+          "185.199.109.153"
```
Comment (iostpa) [inline domains/s1ow.json:13]: You forgot one A record.
```suggestion
          "185.199.108.153",
          "185.199.110.153",
          "185.199.109.153",
          "185.199.111.153"
```

**Review by iostpa — CHANGES_REQUESTED** [review]

**Review by iostpa — APPROVED** [review]
Seems good to me now

**Pull request merged**